sky_alert/endpoint.py

```diff
@@ -7,6 +14,27 @@
 def healthz():
     return 'OK'
 
-@app.get('/sunrise')
-def sunrise():
-    return '06:00'
\ No newline at end of file
+@app.get('/sundata')
```

Comment:
what do you think about using a more descriptive name, including the API that we're calling? Such as 
```suggestion
@app.get('/openweather_sun_data')
```